trl/trainer/sft_trainer.py

```diff
@@ -144,7 +144,7 @@ def __init__(
 
             if callbacks is None:
                 callbacks = [PeftSavingCallback]
-        elif not isinstance(model, PreTrainedModel):
+        elif not isinstance(model, PreTrainedModel) and not isinstance(model, PeftModel):
```

Comment:
```suggestion
        elif isinstance(model, str):
```

Reply:
what about

```python
elif not isinstance(model, (PretrainedModel, PeftModel):
```
which is the same thing but more compact I guess. 

`str` fails on `pathlib.Path` to a local save of a pretrained model, though I don't know if that's supported to begin with

Reply:
but I'm fine with that too

Reply:
I see, can you change it with the suggesiton you made then ? 🙏 Then we can merge I think!

Reply:
ok I pushed the suggestion